obj_gen.cpp

```diff
@@ -530,9 +530,9 @@ imported_keylist::imported_keylist(const char *filename)
 imported_keylist::~imported_keylist()
 {
     while (!m_keys.empty()) {
-        free(m_keys.front());
-        m_keys.erase(m_keys.begin());
-    }
+	free(m_keys.front());
+    }    
```

Comment:
I'm not sure it will work... we are not removing any element from the array... so `m_keys.empty()` always return false.

Reply:
Sorry for that mistake.  There must be something wrong with my brain yesterday.
I just change the code style with a for loop.